apps/snitch_core/lib/core/data/model/line_item.ex

```diff
@@ -55,6 +56,21 @@ defmodule Snitch.Data.Model.LineItem do
     Enum.map(line_items, &set_price_and_total(&1, unit_selling_prices))
   end
 
+  @doc """
+  Returns the item total for given `line_items`.
+
+  If the list is empty, the call is delegated to `MoneyTools.zero/1`.
+  """
+  @spec compute_total([LineItem.t()]) :: Money.t()
+  def compute_total([]), do: MoneyTools.zero!()
```

Comment:
Can this be done using `defdelegate`? just a suggestion

Reply:
No,
1. We're trying to match the empty list, you can't specify args in a `defdelegate` macro.
2. Moreover, the `spec` of both functions doesn't match, a requirement for `defdelegate`.

Reply:
ohkk